cmd/infra/ibmcloud_powervs/create.go

```
@@ -136,6 +149,8 @@ func (infra *Infra) setupInfra(option *CreateInfraOptions) (err error) {
 	}()
 
 	log.Log.Info("Setup infra started")
+	infra.ResourceGroupID, err = getResourceGroupID(option.ResourceGroup)
```

error is not handled properly

---

error handled